app/src/main/java/by/evgen/an11pazniakhomework14/homework15/data/Vodka.kt

```diff
@@ -0,0 +1,31 @@
+package by.evgen.an11pazniakhomework14.homework15.data
+
+import by.evgen.an11pazniakhomework14.homework15.util.tax
+
+open class Vodka(
+    override val name: String = "Vodka",
+    override val alco: Int = 40,
+    override val volume: Int = 50
+) :
+    Spirits(name, alco, volume) {
+
+    open var price: Int = 10
+        set(price) {
+            if ((price > 10) && (price < 100)) {
+                field = price
+            }
+        }
+        get() {
+            return (field * tax).toInt()
+        }
+
```

Comment:
Add only one empty string after close body function

Reply:
Done, thank you.